common_utility/application/handlers/upload_handler.py

```diff
@@ -0,0 +1,52 @@
+import base64
+from uuid import uuid4
+
+from common.constant import StatusCode
+from common.exception_handler import exception_handler
+from common.logger import get_logger
+from common.utils import generate_lambda_response, validate_dict
+from common_utility.application.upload_service import UploadService
+from common_utility.config import SLACK_HOOK, NETWORK_ID
+from common_utility.constants import UPLOAD_TYPE_DETAILS, ALLOWED_CONTENT_TYPE, FILE_EXTENSION, TEMP_FILE_DIR
+from common_utility.exceptions import EXCEPTIONS, InvalidContentType, BadRequestException
+
+logger = get_logger(__name__)
+
+
+@exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger, EXCEPTIONS=EXCEPTIONS)
+def upload_file(event, context):
```

Comment:
As discussed with @pratik-vii Utility handler can be made generic. It does not have to know org or service level details.